style.css

```diff
@@ -0,0 +1,306 @@
+}
+
+section::before 
+{
+  content: "";
+  position: absolute;
+
+  inset: 0 0;
+  width: 50%;
+  min-height: 100vh;
+
+  background: #9d75a8;
+}
+section .container 
+{
+  position: relative;
+  min-width: 1100px;
+  min-height: 550px;
+
+  z-index: 1000;
+  display: flex;
+}
+section .container .contactinfo 
+{
+  position: absolute;
+  top: 40px;
+  width: 350px;
+  height: calc(100% - 80px);
+  padding: 40px;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  background: #1a0633;
+  z-index: 1;
+  box-shadow: 0px 20px 20px rgba(0, 0, 0, 0.904);
+}
+section .container .contactinfo h2 
+{
+  color: rgb(242, 161, 12);
+  font-size: 24px;
+  font-weight: 500;
+}
+section .container .contactinfo .info 
+{
+  position: relative;
+  margin: 20px 0;
+}
+section .container .contactinfo .info li 
+{
+  position: relative;
+  margin: 20px 0;
+  
+  display: flex;
+  align-items: flex-start;
+
+  cursor: pointer;
+  list-style: none;
+}
+
+section .container .contactinfo .info li span:nth-child(1) 
+{
+  width: 30px;
+  min-width: 30px;
+}
+section .container .contactinfo .info li span:nth-child(1) img 
+{
+  width: 100%;
+
+  filter: invert(1);
+  opacity: 0.5;
+}
+section .container .contactinfo .info li span:nth-child(2) 
+{
+  font-weight: 300;
+  color: white;
+  opacity: 0.5;
+
+  margin-left: 10px;
+}
+section .container .contactinfo .info li:hover span:nth-child(1) img,
+section .container .contactinfo .info li:hover span:nth-child(2) 
+{
+  opacity: 1;
+}
+section .container .contactinfo .sci 
+{
+  display: flex;
+  position: relative;
+  align-self: center;
+}
+section .container .contactinfo .sci li 
+{
+  list-style: none;
+  margin: 0 8px;
+}
+section .container .contactinfo .sci li a 
+{
+  text-decoration: none;
+}
+section .container .contactinfo .sci li a img 
+{
+  width: 30px;
+
+  filter: invert(1);
+  opacity: 0.5;
+}
+section .container .contactinfo .sci li:hover a img 
+{
+  opacity: 1;
+}
+section .container .contactForm 
+{
+  position: absolute;
+  padding: 70px 50px;
+  margin-left: 150px;
+  padding-left: 250px;
+  width: calc(100% - 150px);
+  height: 100%;
+
+  background: white;
+  box-shadow: 0 30px 30px rgba(0, 0, 0, 0.978);
+}
+section .container .contactForm h2 
+{
+  font-weight: 300;
+  font-size: 24px;
+  color: #1a0633;
+}
+section .container .contactForm .formBox 
+{
+  position: relative;
+  padding-top: 30px;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+section .container .contactForm .formBox .inputBox 
+{
+  position: relative;
+  margin-bottom: 35px;
+}
+section .container .contactForm .formBox .inputBox.w50 
+{
+  width: 47%;
+}
+section .container .contactForm .formBox .inputBox.w100 
+{
+  width: 100%;
+}
+section .container .contactForm .formBox .inputBox input 
+{
+  width: 100%;
+  padding: 5px 0;
+}
+section .container .contactForm .formBox .inputBox input,
+section .container .contactForm .formBox .inputBox textarea 
```

Comment:
try to write in short don't target element by it's parent if not necessary it will only create difficulty for other developer during reading.

Reply:
ok ill take care of it bhai..